erigon-lib/txpool/fetch.go

```diff
@@ -187,18 +186,21 @@ func (f *Fetch) receiveMessage(ctx context.Context, sentryClient sentry.SentryCl
 			return err
 		}
 		if req == nil {
+			f.logger.Warn("[txpool.receiveMessage]", "req nil")
 			return nil
 		}
+		f.logger.Info("[txpool.receiveMessage]", "req", req)
```

Comment:
requests can be very big. and binary data can be printed useful way - only by fmt.Sprintf("%x")

<img width="1169" alt="Screenshot 2024-07-14 at 16 37 54" src="https://github.com/user-attachments/assets/1c859979-7166-4e7a-b817-48f6d2aeef82">

Reply:
removed because it's useless for majority of situations